database/migrations/2021_06_08_093253_drop_unique_to_users.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class DropUniqueToUsers extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        // userテーブルのunique制約を削除
+        // 退会後同じメールアドレスで登録できる様にする
+        Schema::table('users', function (Blueprint $table) {
+            // ユーザーテーブルからユニークキーを削除
+            $table->dropUnique('users_email_unique');
+            $table->dropUnique('users_username_unique');
```

Comment:
参考サイトは名前のカラム名が `username` みたいなんですが
私たちの今回のテーブルでは `name` を使っているので nameに修正しましょう！:eyes:
```suggestion
            $table->dropUnique('users_name_unique');
```